### ISSUE TITLE
Network: Check out ghostbsd/networkmgr

### ISSUE THREAD

**Issue by probonopd** [helloSystem/Utilities#70]
https://github.com/ghostbsd/networkmgr/ by @ericbsd is very similar to what our Network utility is supposed to do (one day)
Can we make a Qt frontend?

**Comment by ericbsd** [comment]
Yeah, I think we can, the only thing we will need to find how we will implement both, I know there is interest for ncurse also.

**Comment by AndreasTNT** [comment]
Looks like [trayicon.py](https://github.com/ghostbsd/networkmgr/blob/master/src/trayicon.py) is the only file which depends on gtk.

**Comment by ericbsd** [comment]
True

**Comment by probonopd** [comment]
I have started the work, pending
https://github.com/ghostbsd/networkmgr/issues/43
